libs/components/forms/src/lib/modules/checkbox/checkbox.component.spec.ts

```diff
@@ -22,6 +22,8 @@ import { SkyIdService, SkyLogService } from '@skyux/core';
 
```

Comment:
checkbox group implementation?

Reply:
since checkbox group is new it is required by Angular, there is literally no way for them to use it without setting label text